actionpack/CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+*   Add `-g` and `-c` (shorts for _grep_ and _controller_ respectively) options
+    for `rake routes` task. This options returns the url `name`, `verb`, `path`
```

Comment:
We only ever want to refer to `bin/rake` and not just `rake`. Please update this.